cognite/neat/core/data_classes/transformation_rules.py

```diff
@@ -293,13 +299,30 @@ def convert_values(cls, values: dict):
             values["Property"] if isinstance(values["Property"], URIRef) else URIRef(namespace[values["Property"]])
         )
 
-        values["Value"] = cls.get_value(values["Value"], prefixes)
-
-        if not isinstance(values["Value"], URIRef):
-            datatype = (
-                XSD.integer if cls.isint(values["Value"]) else XSD.float if cls.isfloat(values["Value"]) else XSD.string
-            )
-            values["Value"] = Literal(values["Value"], datatype=datatype)
+        # this is safety as when we read from Excel sheet, the value type can be various
+        values["Value"] = str(values["Value"])
+
+        if isinstance(values["Value"], str):
```

Comment:
Since you are forcing string, will not this conditions always be true?

Reply:
Good catch! I've forgot to remove that line!

Removed now :)